docs/_static/nuts-auth.yaml

```diff
@@ -160,6 +160,33 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/AccessTokenRequestFailedResponse"
+  /auth/accesstoken/verify:
+    head:
+      operationId: verifyAccessToken
+      summary: |
+        Verifies the access token given in the Authorization header (as bearer token). If it's a valid access token issued by this server, it'll return a 200 status code.
```

Comment:
Ah, so this endpoint does not intent to implement RFC7662 (https://tools.ietf.org/html/rfc7662) but is sort of a helper API for proxies. I think that's a good solution vs. implementing the RFC in a disputed way.